documentation/installation.md

```diff
@@ -4,9 +4,20 @@
 
 The Kyverno policy engine runs as an admission webhook and requires a CA-signed certificate and key to setup secure TLS communication with the kube-apiserver (the CA can be self-signed). 
 
+## Install Kyverno using Helm
+
+```sh
+## Add the nirmata Helm repository
+ helm repo add kyverno https://nirmata.github.io/kyverno/helm-charts/
+
+## Install the kyverno helm chart
+helm install --name my-release --namespace kyverno nirmata/kyverno
+
+```
+
```

Comment:
Missing header, "Install Kyverno using YAMLs". Option 1 and Option 2 are sub-sections of this.